tests/gem5/suite.py

```diff
@@ -45,7 +45,7 @@
 from testlib.suite import TestSuite
 from testlib.helper import log_call
 from testlib.configuration import constants, config
-from .fixture import TempdirFixture, Gem5Fixture, VariableFixture
+from .fixture import TempdirFixture, Gem5KconfigFixture, VariableFixture
```

Comment:
Do we still need `Gem5Fixture` for anything? Doesn't `Gem5KconfigFixure` supercede it? If we don't then please get rid `Gem5Fixture`?.

Reply:
Changed back to the Gem5Fixture